src/services/data_model_engine.js

```diff
@@ -213,16 +213,19 @@ export default class DataModelEngine {
     await this.entityRepository.discardBundling(bundleStubId);
   }
 
-  async uploadAcceptedBundleCandidates(uploadProgressCallbacks = {success: async () => {}, fail: async () => {}}) {
+  async uploadAcceptedBundleCandidates(uploadProgressControls = {success: async () => {}, fail: async () => {}, stopOnFail: false}) {
```

Comment:
IMO: As this method is only used once, and with stopOnFail set to true I would argue we should remove the parameter and assume it's true.